packages/sanes-browser-extension/src/routes/restore-wallet/test/operateRestoreWallet.ts

```diff
@@ -71,9 +71,6 @@ export const submitRecoveryPhraseE2E = async (
 };
 
 export const travelToRestoreWalletStep = async (page: Page): Promise<void> => {
-  await page.click("button:nth-of-type(1)");
-  await findRenderedE2EComponentWithId(page, UNLOCK_ROUTE);
-
-  await page.click("a:nth-of-type(1)");
+  await page.click("button:nth-of-type(2)");
```

Comment:
Can we use an ID here too because this button has a different number depending on the `hasStoredPersona` state